pkg/controller/deactivation/deactivation_controller_test.go

```diff
@@ -250,7 +250,7 @@ func TestReconcile(t *testing.T) {
 						require.True(t, states.Deactivating(userSignupFoobar))
```

Comment:
This should be set to false if I'm not mistaken.

Reply:
Correct, with the changes to toolchain-common this should now be false.  Updated.